src/main/java/com/epam/idea/rest/config/SocialConfig.java

```diff
@@ -35,6 +35,10 @@
 	public static String facebookClientSecret = "1dc9045cb99ae97b3686cb10648b29b8";
 	public static String vkClientId = "4960267";
 	public static String vkClientSecret = "uu1iDGVhSi8h6EGsL0Gm";
+	public static String facebookClientIdLocal = "1611494735787495";
+	public static String facebookClientSecretLocal = "03c8a369164ba6f44252496c50c384e6";
+	public static String vkClientIdLocal = "4968709";
+	public static String vkClientSecretLocal = "bRqnZIl4qguZ7xCIWNF4";
```

Comment:
Maybe this constants we will get from settings file, or database? Because if this data will change, we need to recompile app, this is bad idea.